src/types/common.ts

```diff
@@ -9,6 +9,27 @@ export const MeResponseSchema = z.object({
 });
 export type MeResponse = z.infer<typeof MeResponseSchema>;
 
+// Response schema for `/workspace` endpoint
+export const WorkspaceResponseSchema = z.object({
+  id: z.string(),
+  industry: z.string(),
+  isCompaniesEnabled: z.boolean(),
+  isClientDirectSignUpEnabled: z.boolean(),
+  logOutUrl: z.string(),
+  brandName: z.string(),
+  squareIconUrl: z.string(),
+  fullLogoUrl: z.string(),
+  squareLoginImageUrl: z.string(),
+  socialSharingImageUrl: z.string(),
+  colorSidebarBackground: z.string(),
+  colorSidebarText: z.string(),
+  colorAccent: z.string(),
+  font: z.string(),
+  metaTitle: z.string(),
+  metaDescription: z.string(),
```

Comment:
so none of these are optional/nullable fields? Please test in workspace if any of these fields may not come.

Reply:
Fixed according to type definition in SDK
![image](https://github.com/copilot-platforms/auto-responder/assets/59971845/bf9077c2-dfdf-44bb-b53d-ddcc5cdce38c)